### PR TITLE
Feature : Sentry Integration + Text about the boba discounted fee.

### PR DESCRIPTION
**Pull request by sk-enya**
**Why ?**
https://github.com/bobanetwork/boba/issues/200

**How ?**
By integrating the sentry with gateway.
https://www.npmjs.com/package/@sentry/react
https://docs.sentry.io/platforms/javascript/guides/react/

**Changes**
- Sentry integration on gateway along with error handling in createActions.
- Avoid sending the sentry events on local server.
- Removed warnings!
- Added text besides the fee switch section to show using boba is discounted.

### CODE REVIEW

**Review by InoMurko — APPROVED** [review]
@Boyuan-Chen @souradeep-das @trangtran-enya wdyt?

File: packages/boba/gateway/src/actions/createAction.js [hunk 1]
```diff
@@ -30,13 +32,15 @@ export function createAction (key, asyncAction) {
         let errorMessage = JSON.parse(response)
         dispatch({ type: `UI/ERROR/UPDATE`, payload: errorMessage.error.message })
         dispatch({ type: `${key}/ERROR` })
+        Sentry.captureMessage(errorMessage.error.message);
```
Comment (souradeep-das) [inline packages/boba/gateway/src/actions/createAction.js:35]: does this capture all non-approved tx errors? do we want to capture these?

Comment (sk-enya) [reply packages/boba/gateway/src/actions/createAction.js:35]: Currently sentry is setup to collect all the errors and crash, I would like to know if we have exclude some errors from capturing.

@souradeep-das can you please help me with same.

Comment (souradeep-das) [reply packages/boba/gateway/src/actions/createAction.js:35]: I was wondering if capturing erc20 permit errors will be useful for us... but on second thoughts I think capturing everything would be better 👍 , we could always filter

**Review by souradeep-das — APPROVED** [review]

**Review by boyuan-chen — APPROVED** [review]
LGTM

File: packages/boba/gateway/src/components/mainMenu/feeSwitcher/FeeSwitcher.js [hunk 2]
```diff
@@ -118,13 +114,22 @@ function FeeSwitcher() {
...
+        <HelpOutline sx={{ opacity: 0.65 }} fontSize="small" />
+      </Tooltip>
+      <Typography variant="body2">Fee</Typography>
+    </S.FeeSwitcherWrapper>
   }
 
-  if (layer !== 'L2') {
-    return null
-  }
+  // if (!accountEnabled) {
```
Comment (InoMurko) [inline packages/boba/gateway/src/components/mainMenu/feeSwitcher/FeeSwitcher.js]: @sk-enya pls delete commented out code!

**Comment by InoMurko** [comment]
@sk-enya to make @java-spaces job easier-let's keep the PR description in form of: 
Why? (What was the issue)
How? (How was it resolved)

Tnx!

**Review by InoMurko — CHANGES_REQUESTED** [review]

**Comment by InoMurko** [comment]
Please also reference the ticket your PR is closing: https://github.com/bobanetwork/boba/issues/200

And close it after it's done! Tnx!

File: packages/boba/gateway/src/components/mainMenu/feeSwitcher/FeeSwitcher.js (hunk 2 above)
Comment (sk-enya) [reply packages/boba/gateway/src/components/mainMenu/feeSwitcher/FeeSwitcher.js]: Sure

**Review by InoMurko — APPROVED** [review]
please merge after cleanup and pr description!

File: packages/boba/gateway/src/actions/createAction.js (hunk 1 above)
Comment (sk-enya) [reply packages/boba/gateway/src/actions/createAction.js:35]: Okay then no changes required we are capturing everything as of now.

**Pull request merged**